Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -16,6 +16,37 @@ import XCTest
 @testable import TensorFlow
 
 final class LayerTests: XCTestCase {
+    func testSequential() {
+        withRandomSeedForTensorFlow((12345, 12345)) {
+            let inputSize = 2
+            let hiddenSize = 4
+            var model = Sequential {
+                Dense<Float>(
+                    inputSize: inputSize,
+                    outputSize: hiddenSize,
+                    activation: sigmoid,
+                    weightInitializer: glorotUniform())
+                Dense<Float>(
+                    inputSize: hiddenSize,
+                    outputSize: 1,
+                    weightInitializer: glorotUniform())
```

Comment:
Could you test with different layer types?

Reply:
Done.